src/main/java/ai/datahunters/md/server/analytics/AnalyticsHandler.java

```diff
@@ -0,0 +1,25 @@
+package ai.datahunters.md.server.analytics;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.ExecutionException;
+
+@AllArgsConstructor
+@Component
+public class AnalyticsHandler {
+
+    private AnalyticsService service;
+
+    public ResponseEntity getAnalytics()  {
+        try {
+            return ResponseEntity.ok(new AnalyticsData(service.photoCount()));
+        }  catch (ExecutionException e) {
+            return ResponseEntity.ok("{" +
```

Comment:
Instead of building exception halding methods everywhere we can use exception handlers from spring, here instead of returning solr exception, please create your own custom exception and throw it here in case of exectuion exception (if this is really the solr exception)

Reply:
I don't see the point of custom exception; if we're going to generate error status page/response using @ControllerAdvice (as I assume), I'll just let all exceptions to be passed thru Spring.